examples/matmul/run.cpp

```diff
@@ -7,6 +7,10 @@
 #include "gpu.h" // createContext, createTensor, createKernel, dispatchKernel,
                  // wait, resetCommandBuffer, toCPU
 
+#ifndef M_PI
```

Comment:
IIRC this is because `_USE_MATH_DEFINES` is not set in MSVC builds -- maybe a better approach is to define that value as a preprocessor directive in CMakeLists and Makefile.